src/toil/test/docs/scripts/tutorial_cwlexample.py

```diff
@@ -25,7 +26,7 @@ def runQC(job, cwl_file, cwl_filename, yml_file, yml_filename, outputs_dir, outp
 
 
 if __name__ == "__main__":
-    options = Job.Runner.getDefaultOptions("./toilWorkflowRun")
+    options = Job.Runner.getDefaultOptions(tempfile.mkdtemp("tutorial_cwlexample")+os.sep+"toilWorkflowRun")
```

Comment:
Even though we clean the workflow directory up, won't we leave behind the `mkdtemp()` directories here? Maybe we should use the `mkdtemp()` directory as the workflow directory to make sure it gets deleted, instead of using another directory inside it.